backend/btrixcloud/operator/baseoperator.py

```diff
@@ -78,11 +78,12 @@ def compute_crawler_resources(self):
         max_crawler_memory_size = 0
         max_crawler_memory = os.environ.get("MAX_CRAWLER_MEMORY")
         if max_crawler_memory:
-            max_crawler_memory_size = int(parse_quantity(max_crawler_memory_size))
+            max_crawler_memory_size = int(parse_quantity(max_crawler_memory))
 
         self.max_crawler_memory_size = max_crawler_memory_size or crawler_memory
 
-        print("max crawler memory size", self.max_crawler_memory_size)
+        print(f"max crawler memory size: {self.max_crawler_memory_size}")
+        print()
```

Comment:
```suggestion
```

Reply:
This was to add a newline in the logs :)